builtins/src/fs_meta.rs

```diff
@@ -40,6 +41,25 @@ fn cd_expand_all_dots(cd: String) -> String {
     }
 }
 
+/// Usage: (sleep milliseconds) -> nil
+///
+/// Sleep for *at least* the provided milliseconds (must be a positive integer),
+/// otherwise function will no-op.
+///
+/// Section: system
```

Comment:
not sure about this section name

Reply:
yeah. if we get `time` moved in I can leave the test for sleep in which would be nice.